notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb

```diff
@@ -75,7 +75,7 @@
         "id": "dbab58d4ae1a"
```

Comment:
<p>combine these code cells. Do print(dfX) instead of dfX</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='105'/>

Reply:
<p>done</p>